tests/commands/test_edit.py

```diff
@@ -0,0 +1,300 @@
+import os
+import pytest
```

Comment:
Nit: convention[1] states that imports of 3rd-party modules should come after standard-lib imports.

Not a blocker, just FYI.

[1] https://peps.python.org/pep-0008/#:~:text=Imports%20should%20be%20grouped%20in,Local%20application%2Flibrary%20specific%20imports.